### ISSUE TITLE
Update link to HTML spec from WCAG section 7

### ISSUE THREAD

**Issue by plannero** [w3c/wcag#2483]
The deep link [HTML 5.2 Autofill field section](https://www.w3.org/TR/html52/sec-forms.html#sec-autofill) meant to reference the table containing autofill values (with examples) from section 7 of WCAG (both WCAG 2.1 and 2.2) is now redirecting to another url, which does not have a "#sec-autofill" destination. 

So the link is much less useful now.

I believe the link should be changed to https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-field
